lib/msal-core/docs/cdn.md

```diff
@@ -0,0 +1,145 @@
+# CDN Usage of MSAL.js
+
+In addition to npm, `msal` can be consumed from Microsoft-hosted CDNs.
+
+## Best Practices
+
+* Use the latest version of MSAL.js.
+* Use the minified build in production, unminified build for development.
+* Use the Microsoft CDN instead of third-party CDNs.
+* Use the CDN region nearest to your users.
+* Use the `async` or `defer` attributes to not block page rendering.
+* Use the `integrity` attribute to ensure integrity of CDN builds.
+* IE11 support requires a Promise polyfill ([more information](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/samples/msal-core-samples/VanillaJSTestApp/app/ie11-sample/README.md) on IE11 support).
+
+## Basic Usage
+
+```html
+<script type="text/javascript" src="https://alcdn.msauth.net/lib/1.4.1/js/msal.min.js"></script>
```

Comment:
Trying to minimize the number of places where we would need to update with the latest version number.

Reply:
We could keep the one here generic, and update the one in the README with the latest.